content/src/ui/Component.SessionUnavailable.json

```diff
@@ -0,0 +1,11 @@
+{
+  "en": {
+    "description": "The session you're looking for is not available anymore."
+  },
+  "sv": {
+    "description": "The session you're looking for is not available anymore."
+  },
+  "pt": {
+    "description": "The session you're looking for is not available anymore."
```

Comment:
It's better to not duplicate keys in `sv` and `pt`. Then it's easier to see in the CMS what needs translations as they will be empty.

Reply:
Ah, gotcha, makes sense 👍